glean-core/rlb/src/lib.rs

```diff
@@ -221,6 +226,29 @@ pub fn initialize(cfg: Configuration, client_info: ClientInfoMetrics) {
         with_glean_mut(|glean| {
             let state = global_state().lock().unwrap();
 
+            // The debug view tag might have been set before initialize,
+            // get the cached value and set it.
+            if let Some(tag) = PRE_INIT_DEBUG_VIEW_TAG.get() {
+                let lock = tag.try_lock();
+                if let Ok(ref debug_tag) = lock {
+                    glean.set_debug_view_tag(debug_tag);
+                }
+            }
+            // The log pings debug option might have been set before initialize,
+            // get the cached value and set it.
+            let log_pigs = PRE_INIT_LOG_PINGS.load(Ordering::SeqCst);
+            if log_pigs {
+                glean.set_log_pings(log_pigs);
+            }
```

Comment:
Hm, I guess it's not wrong to do the check here, because if you flip back and forth the `log pings` setting before enable you really only need to pass through a `true` value.

But not having the `if log_pings` would be just as correct.

Reply:
You are correct!